### PR TITLE
Modify the QR code of the community group to remove the qq group information

### PR DESCRIPTION
**Pull request by peacewong**
### What is the purpose of the change
1. Modify the QR code of the community group to remove the qq group information
2. add codecov
3. Modify the Compile and deploy chapter

### CODE REVIEW

File: README_CN.md [hunk 1]
```diff
@@ -63,6 +64,17 @@ Linkis 自2019年开源发布以来，已累计积累了700多家试验企业和
 请前往[Linkis releases 页面](https://github.com/apache/incubator-linkis/releases) 下载Linkis 的已编译版本或源码包。
 
 # 编译和安装部署
+```shell
+
+## 后端编译
```
Comment (casionone) [inline README_CN.md:69]: Whether to differentiate the OS?
For Win cmd  should be used as 
```
mvnw.cmd -N install
mvnw.cmd clean install -Dmaven.javadoc.skip=true -Dmaven.test.skip=true
```

Comment (peacewong) [reply README_CN.md:69]: ok, thanks.

**Review by CCweixiao — COMMENTED** [review]
LGTM.

**Review by CCweixiao — COMMENTED** [review]
LGTM.

**Review by CCweixiao — APPROVED** [review]

**Pull request merged**